### PR TITLE
Add --log-level support

### LINKED ISSUE #79: Add -l --log-level
## Description:
- Add to Bluebell

- References #xx

### PR DESCRIPTION
**Pull request by bartr**
## Type of PR

- [ ] Documentation changes
- [X] Code changes
- [X] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
- Added support for --log-level
- converted --auth-type to enum for simplicity

## Issues Closed or Referenced

- Closes #79

### CODE REVIEW

**Review by mag16 — APPROVED** [review]

File: src/app/Middleware/RequestLogger/loggerOptions.cs [hunk 1]
```diff
@@ -9,6 +9,6 @@ public class LoggerOptions
         public bool Log3xx { get; set; } = true;
         public bool Log4xx { get; set; } = true;
         public bool Log5xx { get; set; } = true;
-        public double TargetMs { get; set; } = 250;
+        public double TargetMs { get; set; } = 1000;
     }
```
Comment (aflinchb) [inline src/app/Middleware/RequestLogger/loggerOptions.cs:13]: Was this change done to reduce volume of logs?

File: src/tests/ParamTest.cs [hunk 2]
```diff
@@ -25,21 +25,25 @@ public async Task AppMainTest ()
 
             args = new string[] { "-k", "heliumtest-kv", "-a", "FOO" };
             i = await App.Main(args);
-            Assert.Equal(1, i);
+            Assert.NotEqual(0, i);
 
-            Assert.Equal(2, App.CombineEnvVarsWithCommandLine(null).Count);
+            Assert.Equal(4, App.CombineEnvVarsWithCommandLine(null).Count);
```
Comment (aflinchb) [inline src/tests/ParamTest.cs:31]: (For my understanding) - why is this 4?

**Review by aflinchb — APPROVED** [review]
Approved with a few questions for my understanding.

File: src/app/Middleware/RequestLogger/loggerOptions.cs (hunk 1 above)
Comment (bartr) [reply src/app/Middleware/RequestLogger/loggerOptions.cs:13]: It was set at 800 in the constructor which I removed.  250 seemed a little aggressive, so I went with 1 second. You can change it in the constructor when you consume the package.

File: src/tests/ParamTest.cs (hunk 2 above)
Comment (bartr) [reply src/tests/ParamTest.cs:31]: because we added --log-level which creates 2 entries (--log-level and value)

**Pull request merged**